lib/screens/calc_screen.dart

```diff
@@ -0,0 +1,175 @@
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '2',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '3',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: 'x',
+                          color: 'blue',
+                        ),
+                      ],
+                    ),
+                    const SizedBox(height: 16),
+                    Row(
+                      children: const [
+                        Button(
+                          text: '4',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '5',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '6',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '-',
+                          color: 'blue',
+                        ),
+                      ],
+                    ),
+                    const SizedBox(height: 16),
+                    Row(
+                      children: const [
+                        Button(
+                          text: '7',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '8',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '9',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '+',
+                          color: 'blue',
+                        ),
+                      ],
+                    ),
+                    const SizedBox(height: 16),
+                    Row(
+                      children: const [
+                        Button(
+                          text: '.',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '0',
+                          color: 'white',
+                        ),
+                        SizedBox(width: 16),
+                        Button(
+                          text: '<[]',
```

Comment:
jeżeli chodzi o ten symbol to możesz go wziąć z figmy i zapisać jako SVG i potem paczke musisz mieć, żeby go zaimportować, ale to sobie znajdziesz